.github/workflows/e2e_test.yml

```diff
@@ -102,6 +103,28 @@ jobs:
             profile_step=3 \
             max_steps=15
 
+      - name: Run Llama 3.0 8B (2 slice)
```

Comment:
how can we avoid hard coded model names? can we have a yaml file with all candidate models that can subscribe to the multi-slice solution?

Reply:
In principle all models could be tested with multi-slice but we only have 256 chips in `us-central2-b` at the moment (64 v6e-4 => 32 2-slice runs) and we'd need to account for parallel PRs and test runs etc. I filed https://github.com/AI-Hypercomputer/torchprime/issues/164 that should at least make it simpler.